classes/player/player.gd

```diff
@@ -420,6 +420,8 @@ func adjust_swim_x() -> void:
 	vel.x += (swim_adjust - vel.x) * FPS_MOD
 
 
+const SPRAY_ORIGIN = Vector2(-9, 4)
```

Comment:
```suggestion
const SPRAY_ORIGIN = Vector2(-9, 6)
```
This allows it to fit snugly into the nozzle without being visible above it.

Reply:
Ah yep, that does look better!